cmd/rootCmd.go

```diff
@@ -83,9 +83,19 @@ var (
 		},
 	}
 
-	pvcBrowseLocalPort int
-	pvcBrowseCmd       = &cobra.Command{
-		Use:   "browse [PVC name]",
+	browseLocalPort int
+	browseCmd       = &cobra.Command{
```

Comment:
We should mark this command as deprecated and print a message to use `browse pvc` instead . 
https://github.com/spf13/cobra/blob/371ae25d2c82e519feb48c82d142e6a696fd06dd/command.go#L102

Reply:
Done! ✅  This is how the `stdout` looks now.
```
> ./kubestr browse test-pvc1 -n file-level-restore -v test-snapshotclass
Command "browse" is deprecated, we recommend you to use command 'browse pvc' instead. Command 'browse pvc' will support newer updates for browsing through the contents of a PVC.
Taking a snapshot
Creating the file browser application.
Forwarding the port.
Port forwarding is ready to get traffic. visit http://localhost:8080/
^CStopping port forward
Cleaning up resources
```